.travis.yml

```diff
@@ -6,10 +6,10 @@ virtualenv:
 before_install:
   - sudo sh -c "echo deb http://www.openfoam.org/download/ubuntu precise main > /etc/apt/sources.list.d/openfoam.list"
   - sudo apt-get update -qq
-  - sudo apt-get install -y --force-yes openfoam222
+  - sudo apt-get install -y --force-yes openfoam230
```

Comment:
Making automatic testing also for 240 is a little bit tricky while Ubuntu 12.04LTS dont have prebuild debian packages to 240. So we should make debian package and make it available somewhere (Ioannis suggested launchpad) to be avle to load in travis.  I suppose that we are anyway going towards Ubuntu 14.04 and also OF3.0, so i would not concider doing that now.